rj_param_utils/CMakeLists.txt

```diff
@@ -24,28 +24,15 @@ set(RJ_PARAM_UTILS_DEPS_INCLUDE_DIRS include)
 # ======================================================================
 # Include and Linking
 # ======================================================================
-target_include_directories(rj_param_utils PUBLIC ${RJ_PARAM_UTILS_DEPS_INCLUDE_DIRS})
+target_include_directories(rj_param_utils PRIVATE ${RJ_PARAM_UTILS_DEPS_INCLUDE_DIRS})
 
-target_link_libraries(rj_param_utils PUBLIC ${RJ_PARAM_UTILS_DEPS_LIBRARIES})
+target_link_libraries(rj_param_utils PRIVATE ${RJ_PARAM_UTILS_DEPS_LIBRARIES})
 
 ament_target_dependencies(rj_param_utils PUBLIC rclcpp)
 
-# ======================================================================
-# Testing
-# ======================================================================
-if(BUILD_TESTING)
-  enable_testing()
-
-  add_library(rj_param_utils_testing STATIC)
-  add_subdirectory(testing)
-  target_link_libraries(rj_param_utils_testing PUBLIC rj_param_utils ${GTEST_LIBRARIES})
-endif()
-
 # ======================================================================
 # Packaging
 # ======================================================================
 install(
-  TARGETS rj_param_utils
-  ARCHIVE DESTINATION lib
-  LIBRARY DESTINATION lib
-  RUNTIME DESTINATION bin)
+  TARGETS global_params
+  DESTINATION lib/${CMAKE_PROJECT_NAME})
```

Comment:
Again, you can keep the old install target (and the testing subdirectories above).